client/src/pages/EditProject.js

```diff
@@ -0,0 +1,327 @@
+            Preview
+                    </Button>
+        </div>
+        <div className={classes.addProjectPage}>
+          <Typography variant="h2" align='left'>Edit your project</Typography>
+          <form className={classes.form} onSubmit={this.handleSubmit} >
+
+            <Typography variant="h4">Title</Typography>
+            <TextField
+              name="title"
+              classes={{ root: classes.formLine }}
+              value={title}
+              fullWidth={true}
+              onChange={this.handleInput}
+              type="title"
+              variant="outlined"
+              required
+              error={validation.title.isInvalid}
+              helperText={validation.title.message}
+            />
+
+
+            <Typography variant="h4">Subtitle</Typography>
+            <TextField
+              name="subtitle"
+              classes={{ root: classes.formLine }}
+              value={subtitle}
+              fullWidth={true}
+              onChange={this.handleInput}
+              type="subtitle"
+              variant="outlined"
+            />
+
+            <Typography variant="h4">Description</Typography>
+            <TextField
+              name="description"
+              multiline
+              rows="5"
+              classes={{ root: classes.formLine }}
+              value={description}
+              fullWidth={true}
+              onChange={this.handleInput}
+              type="description"
+              variant="outlined"
+            />
+
+            <OutlinedSelect
+              useLabel
+              name="industry"
+              labelText="Industry"
+              selectId="industry"
+              setState={this.handleInput}
+              selectName="industry"
+              value={industry}
+            >
+              {
+                industries.map(industry => {
+                  return (
+                    <option
+                      key={industry.id}
+                      value={industry.name}
+                    >
+                      {industry.name}
+                    </option>
+                  )
+                })
+              }
+            </OutlinedSelect>
+            <OutlinedSelect
+              useLabel
+              name="location"
+              labelText="Location"
+              selectId="location"
+              setState={this.handleInput}
+              selectName="location"
+              value={location}
+            >
+              {
+                locations.map(location => {
+                  return (
+                    <option
+                      key={location.id}
+                      value={location.name}
+                    >
+                      {location.name}
+                    </option>
+                  )
+                })
+              }
+            </OutlinedSelect>
+            <UploadImages setImages={this.setImages} showMany={true} images={updatedImages} deleteImage={this.deleteImage} />
```

Comment:
Just a suggestion: Remove the drop shadow on here or if it's a material-ui card, you can do elevation={0}

Reply:
I lightened the shadow